VerilogCodingStyle.md

```diff
@@ -797,6 +797,92 @@ endmodule : foo
 localparam bit ValBaz = 1;  // This comment describes the item to the left.
 ```
 
+It can sometimes be useful to structure the code using header-style comments in
+order to separate different functional parts (like FSMs, the main datapath or
+registers) within a module. In that case, the preferred style is as follows:
+
+```systemverilog
+module foo;
+
+  ////////////////
+  // Controller //
+  ////////////////
+  ...
+
+  ///////////////////////
+  // Main ALU Datapath //
+  ///////////////////////
+  ...
+
+  ///////////////////////////////////////////////////////
```

Comment:
If description is needed, it should be a separate comment below the subheading in my opinion.

Reply:
other opinions? I am fine both with keeping or removing it..

Reply:
```
//////////////////
// Main Heading //
//////////////////
// Description: blah blah
//     extended descriptions.
```

Reply:
ok will remove it